docs/changes/timeseries/17251.perf.rst

```diff
@@ -0,0 +1,2 @@
+Reduction of redundant computation of the the ``cython`` implementation of the Lomb-Scargle periodogram.
```

Comment:
Typo. Also, we generally ask for these changelog fragment to be 80 cols wide at most
```suggestion
Reduction of redundant computation of the ``cython`` implementation of the Lomb-Scargle periodogram.
```